src/main/java/vct/main/Main.scala

```diff
@@ -176,7 +176,7 @@ class Main {
         silver.set("silicon")
       case "silicon" =>
       case "carbon" =>
-        Configuration.checkCarbonRequirements()
+        // Nothing to check for
```

Comment:
This suggests that the silicon case falls through to the carbon case, which is not true. Please add a similar comment to silicon or remove it :)